bin/os-detect

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# detect os platform
+OS_TYPE="$(uname -s)"
+case "${OS_TYPE}" in
+  Linux*) sort_command=sort grep_command=grep;;
+  Darwin*) sort_command=gsort grep_command=ggrep;;
```

Comment:
If these don't exist, we'll need to print a warning to install them!

Reply:
I'm sorry, I was too late to respond to this. based on https://github.com/sirupsen/zk/pull/7/commits/000bbf7fa13ac480a6fa06faa1a30a52068ec3de#r470224199 i think we could print a warning. and then do exit from os-detect file. is it good?